client/Pages/EventTypeCreate/View.elm

```diff
@@ -262,17 +265,21 @@ viewForm model setup =
                 , selectInput formModel
                     FieldCleanupPolicy
                     "Cleanup policy"
-                    (if (getValue FieldCleanupPolicy formModel.values) == cleanupPolicies.compact then
-                        "Log compacted event types MUST NOT contain personal identifiable"
-                            ++ " information in accordance to GDPR. If you plan to store user"
-                            ++ " data permanently, check the legal department of your organization"
-                     else
-                        ""
-                    )
+                    ""
                     Help.cleanupPolicy
                     Optional
                     Enabled
                     cleanupPoliciesOptions
+                , if (getValue FieldCleanupPolicy formModel.values) == cleanupPolicies.compact then
+                    p [ class "dc-p"  ]
+                        [ text "Log compacted event types MUST NOT contain personal identifiable"
```

Comment:
Would it make sense to put this text in some configuration? I imagine that other people who want to deploy nakadi-ui may have different opinions about what should and what should not be allowed to store in log-compacted event types

Reply:
Yes. I thought about it but I don't want to blow up configuration just right now. I would wait for some time to settle it down a bit and ideally have a real use case for this config.

Reply:
ok